pdns/validate.cc

```diff
@@ -763,11 +763,13 @@ static const vector<DNSName> getZoneCuts(const DNSName& begin, const DNSName& en
 
 bool isRRSIGNotExpired(const time_t now, const shared_ptr<RRSIGRecordContent>& sig)
 {
+  // Should use https://www.rfc-editor.org/rfc/rfc4034.txt section 3.1.5 
   return sig->d_sigexpire >= now;
```

Comment:
We have `rfc1982LessThan` in our code base, but I'm not sure it can be used with these types.

Reply:
I tried to implement this, but the 2106 still fails. From a quick look because the record cache cannot handle timestamps beyond 2106, causing trouble finding the root servers. I'd say we leave this for some other time.

Reply:
Fine by me :) Perhaps we can open a new issue unless the validation case is already covered by the existing one?

Reply:
#10000 mentions it.